docs/guides/postman-collection.md

```diff
@@ -0,0 +1,87 @@
+# Postman Collections support
+
+Prism offers a _limited_ support for Postman Collection. The basic workflow of using Prism (both from the CLI and deployed in a Docker container) are fundamentally the same, however there are some known limitations that it's important to keep in mind:
+
+## Authentication
+
+Postman collections support more authentication methods than OpenAPI 2 and 3, although some of these are a specialisation of another method included in their support; this is mostly to drive the UI of their application.
+
+In case the authentication method does not have a corresponding element in the OpenAPI specification, it can ultimately be expressed as a header. In such a case, Prism will "downgrade" according to the following table:
+
+| Postman Collection | Transformation         |
+| ------------------ | ---------------------- |
+| apikey             | apikey                 |
+| bearer             | http + bearer scheme   |
+| digest             | http + digest scheme   |
+| oauth1             | http + bearer scheme   |
+| oauth2             | http + bearer scheme   |
+| awk                | Hardcoded header value |
+| aws                | Hardcoded header value |
+| akamai             | Hardcoded header value |
+| ntlm               | Hardcoded header value |
+
+## Events
```

Comment:
People are not usually looking at their postman collections directly. There are exceptions to this, but most of the time its synced somehow from somewhere and Postman is the editor. With that in mind, I don't think anyone would even know what Events is, let alone care that its not being brought across in Prism.

Reply:
I do not know how much people are using the event — the reason why I brought this up here is because an event can modify a global variable which could change the way a request/response pair is going to behave (if such global variable is used) — clearly Prism cannot detect such change and so it won't respond in the same way.

I'm fine removing it if you confirm, I just thought it might be important for advanced users.

Reply:
Does this still matter if we don't really support global env variables anywhere?

Reply:
Effectively we do not because they're stored in the app. I'll remove it.